sema/ro/creator/strategy/roc_basic.py

```
@@ -0,0 +1,22 @@
+from pathlib import Path
+from sema.ro.creator.api import RocModel, RocStrategy, RocStrategyContext
+
+
+class BasicStrategy(RocStrategyContext, RocStrategy):
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)